src/api/types.ts

```diff
@@ -319,6 +323,8 @@ interface Feedback {
   otherComments: string;
   referralUrl: string;
   product: string;
+  createdAt: string;
+  success: boolean;
```

Comment:
I don't think we need `success` here, could the resolver just use this as a return type? Unless we need to return the whole feedback object

```js
Promise<SuccessResponse>
```

Reply:
its useful for the test to check if feedback has been generated? won't be able to test it otherwise in API

Reply:
I see, in the custom resolver can you add the return type?

`Promise<Feedback>`

Reply:
Actually don't need to add `success: boolean;` here as it already gets it from `SuccessResponse` - not a blocker if we remove this in next PR, so we can get this merged and progress the sprint